prime-router/src/main/resources/db/migration/V10__Update_settings_for_name_templates.sql

```diff
@@ -0,0 +1,62 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ * use VARCHAR(63) for names in organization and schema
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+/*
+ * Update the setting table, changing values to type jsonb instead of json
+ */
+ALTER TABLE setting
+ALTER COLUMN values
+SET DATA TYPE JSONB
+USING values::JSONB;
+
+/*
+ *  Remove the useAphlNamingFormat key
+ */
+UPDATE
+    setting s
+SET
+    values = CASE (s.values::JSONB#>'{translation,useAphlNamingFormat}')
+                 WHEN 'true' THEN
+                     JSONB_SET(
+                             s.values::JSONB,
+                             '{translation}',
+                             JSONB_SET(
+                                     ((s.values::JSONB -> 'translation') - 'useAphlNamingFormat'),
+                                     '{nameFormat}',
+                                     '"aphl"'
+                                 )
+                         )
+                 WHEN 'false' THEN
+                     JSONB_SET(
+                             s.values::JSONB,
+                             '{translation}',
+                             JSONB_SET(
+                                     ((s.values::JSONB -> 'translation') - 'useAphlNamingFormat'),
+                                     '{nameFormat}',
+                                     COALESCE((s.values::JSONB#>'{translation,nameFormat}'), '"standard"')
+                                 )
+                         )
+                 ELSE s.values::JSONB
+        END
+WHERE
+    s.type = 'RECEIVER'
+    AND s.values::JSONB#>'{translation}' IS NOT NULL
+    AND s.is_active = TRUE
+;
+
+UPDATE
+    setting s
+SET
+    values = JSONB_SET(s.values::JSONB, '{translation,nameFormat}', '"standard"')
+WHERE
+    s.type = 'RECEIVER'
+    AND s.values::JSONB#>'{translation,nameFormat}' IS NULL
+    AND s.is_active = TRUE
+;
```

Comment:
This removes the `useAphlNamingFormat`. This is what manipulating JSON looks like in postgres.

Reply:
When you merge this into Staging, can you please do some spot checking of the data to confirm that it worked?   And then again when this goes into Prod.  Thanks